CONTRIBUTING.md

```diff
@@ -1,55 +1,57 @@
 # Lethe
 
-Lethe is an open source and we try to follow the deal.II mentality of having it
-developed in an open accessible environment. Consequently, all scientific
-developments made within Lethe are accessible to everyone even as they are
+Lethe is an open source software, and we try to follow the deal.II mentality of having it
+developed in an accessible environment. Consequently, all scientific
+developments made within Lethe are available to everyone even as they are
 currently being written. Anybody that wishes to use and contribute to a feature
-is welcome to do so.
+is welcomed to do so.
 
-This document aims at establishing guidelines for contributions within
-Lethe.
+This document aims at establishing guidelines for contributions within Lethe.
 
-Lethe has a wiki which is openly accessible on : https://github.com/chaos-polymtl/lethe/wiki
+Lethe has its own documentation (user guide and developer guide) which are openly accessible on: https://chaos-polymtl.github.io/lethe/index.html
```

Comment:
```suggestion
Lethe has its own documentation (user guide and developer guide) which are available on: https://chaos-polymtl.github.io/lethe/index.html
```

Reply:
Bruno suggested to change accessible to available, will let you fight on that one 👀